### DISCUSSION TITLE
Error while saving related inline model

### DISCUSSION THREAD

**Discussion by markosandalj** [jrief/django-admin-sortable2#331] [General]
I'm getting "getattr(): attribute name must be string" erorr when I try to add new relation through inline panel. Everything worked properly when I tried it out locally but once I deployed my app it stopped working.

`/lib/python3.9/site-packages/adminsortable2/admin.py line, line 467, in save_new`

Here is a video with an example: https://vimeo.com/749720112/d4b118c0ba

All my models that I use as inlines have these fields:
```
sort_order = models.PositiveIntegerField(default=0, blank=False, null=False)

class Meta(object):
        ordering = ['sort_order',]
```
        
I'm not sure what else to provide so it will be easier to figure out whats going so please point out if I can add any info that might help.

**Comment by jrief** [comment]
Please use the testapp to describe a procedure on how to reproduce this error.
